frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectPortlet.java

```diff
@@ -383,7 +424,7 @@ private void sendLicenseInfoResponse(ResourceRequest request, ResourceResponse r
     }
 
     private void saveLicenseInfoAttachmentUsages(Project project, User user, Set<String> selectedAttachmentIdsWithPath,
-            Map<String, Set<LicenseNameWithText>> excludedLicensesPerAttachmentIdWithPath) {
+            Map<String, Set<LicenseNameWithText>> excludedLicensesPerAttachmentIdWithPath, List<String> list) {
```

Comment:
Please rename list parameter to meaningful name

Reply:
Done